language/google/cloud/language/syntax.py

```diff
@@ -183,11 +185,18 @@ class Sentence(object):
     :param begin: The beginning offset of the sentence in the original
                   document according to the encoding type specified
                   in the API request.
+
+    :type sentiment: :class:`~google.cloud.language.sentiment.Sentiment`
+    :param sentiment: For calls to
+                      :func:`~google.cloud.language.document.Document.annotate_text`
+                      where ``include_sentiment`` is set to true, this field
+                      will contain the sentiment for the sentence.
```

Comment:
To appease lint, wrap the whole description to a new line (after `:param sentiment:`) and indent by four spaces.